### DISCUSSION TITLE
科技爱好者周刊（第 255 期）：对待 AI 的正确态度 | ruanyf-weekly

### DISCUSSION THREAD

**Discussion by giscus** [plantree/press-comment#17] [General]
# 科技爱好者周刊（第 255 期）：对待 AI 的正确态度 | ruanyf-weekly

阮一峰的技术周刊

https://ruanyf-weekly.vercel.app/weekly/issue-255

**Comment by terminalqo** [comment]
应该不叫**最大可行产品**，而是**最大范围产品**？

**Reply by plantree** [reply]
很符合“安迪-比尔定律”
